ctsit/redcap_zebra_label_printer
# Feature/pull data from form

This PR updates the README and adds config to pull `ptid` and `visit_num` using the project configuration.

**Related issues**
Closes #4 and #5.

**To test**
1. Import the [sample project](https://drive.google.com/drive/folders/1yPW0HAmlW0FyRGJOyX0TFtYFre0-hIja) into a REDCap 14.1.3 local instance
2. Enable the external module globally
3. Enable the external module for the imported sample project
4. Configure the external module in the imported project setting ptid and visit num to the corresponding fields:
  - > <img width="868" alt="image" src="https://github.com/ctsit/tube_label_generator/assets/75623654/60cc3241-449b-4f7d-b4e6-de7a26041d6a">
5. Create some dummy records 
6. Open the __Generate Labels__ page and verify that the dropdown populates with the dummy data entered in step 5
  - > <img width="358" alt="image" src="https://github.com/ctsit/tube_label_generator/assets/75623654/1be063b4-7851-4275-8864-1e2ca17c741c">
